templates/application-ccm-gluekube.yaml

```diff
@@ -0,0 +1,101 @@
+{{- if .Values.kubeadm.enabled -}}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: glueops-ccm-gluekube
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    name: "in-cluster"
+    namespace: glueops-core-ccm
+  project: glueops-core
+  syncPolicy:
+    syncOptions:
+      - CreateNamespace=true
+    automated:
+      prune: true
+      selfHeal: true
+    retry:
+      backoff:
+        duration: 5s
+        factor: 2
+        maxDuration: 3m0s
+      limit: 2
+  source:
+    repoURL: https://helm.gpkg.io/project-template
+    chart: app
+    targetRevision: 0.8.1
+    helm:
+      values: |+
+        appName: "glueops-ccm-gluekube"
+      
+        serviceAccount:
+          create: true
+          name: webhook
+        
+        customResources:
+          - |-
+            apiVersion: metacontroller.k8s.io/v1alpha1
+            kind: DecoratorController
+            metadata:
+              name: lb-service-decorator
```

Comment:
prefix with: `glueops-core-ccm-*`

Reply:
I believe this is a global cluster resource. If it's namespaced then as-is is fine.